### PR TITLE
Developing the remaining basic ui exercises  🚀

### PR DESCRIPTION
**Pull request by jcgt96**
According to the repository where are all the exercises, were completed all the scenarios proposed in the file `basic_ui_exercises.feature`

### CODE REVIEW

File: src/main/features/basic_ui_exercises.feature [hunk 1]
```diff
@@ -29,3 +29,28 @@ Feature: Practicing Basic User Interactions
         And she selects "Option3" in the checkbox
         Then she should see that the "Option1" checkbox is checked
         And she should see that the "Option3" checkbox also is checked
+
+  #@skip
+  Scenario: Displaying a Custom Alert with the Name
+    When she triggers an Alert Dialog with the name "ALEX"
```
Comment (AlexAndradeNuvei) [inline src/main/features/basic_ui_exercises.feature]: Use your name

File: src/main/features/basic_ui_exercises.feature [hunk 2]
```diff
@@ -29,3 +29,28 @@ Feature: Practicing Basic User Interactions
...
+    Then she should see an Alert Dialog with the text "Hello ALEX, share this practice page and share your knowledge"
+
+
+  Scenario: Displaying a Custom Confirm Dialog with a Custom Name
+    When she triggers a Confirmation Dialog for the name "ALEX"
```
Comment (AlexAndradeNuvei) [inline src/main/features/basic_ui_exercises.feature]: Use your name

File: src/main/features/steps/home_page_steps.py [hunk 3]
```diff
@@ -75,3 +75,68 @@ def step_then_should_see_the_checkbox1_is_selected(context):
...
+def step_check_third_price(context, price):
+    assert context.third_value == price
+
+
+# Eigth Scenario
+@when('Verena can see that the "{text}" text box is visible')
+def step_can_see_visible_text(context, text):
+    context.home_page.write_in_text_box(text)
+    context.home_page.show_text_box()
+    assert context.home_page.get_text_box_text() in text
```
Comment (AlexAndradeNuvei) [inline src/main/features/steps/home_page_steps.py:120]: Why asserts on @When?

**Review by AlexAndradeNuvei — CHANGES_REQUESTED** [review]

File: src/main/features/steps/home_page_steps.py (hunk 3 above)
Comment (jcgt96) [reply src/main/features/steps/home_page_steps.py:120]: The step says: `When Verena can see that the "HideShow Example" text box is visible` so I decided make the assertion there to validate that Verena **can** see that the text box is visible and match with the expected text

File: src/main/features/basic_ui_exercises.feature (hunk 2 above)
Comment (jcgt96) [reply src/main/features/basic_ui_exercises.feature]: ✅

File: src/main/features/basic_ui_exercises.feature (hunk 1 above)
Comment (jcgt96) [reply src/main/features/basic_ui_exercises.feature]: ✅

File: src/main/features/steps/home_page_steps.py (hunk 3 above)
Comment (AlexAndradeNet) [reply src/main/features/steps/home_page_steps.py:120]: Ok. I got it. It's necessary.

**Pull request merged**